tests/test_core.py

```diff
@@ -134,11 +157,13 @@ def g():
     results = g()
 
     def match():
-        match_result = next(results)
-        return match_result and match_result.region
+        return next(results)
 
-    result = wait_until(match, stable_secs=2)
-    assert result == stbt.Region(x=4, y=0, width=10, height=2)
+    result = wait_until(match, predicate=lambda x: x and x.region,
+                        stable_secs=2)
```

Comment:
You could ditch `match` and write:

        result = wait_until(g().next, predicate=lambda x: x and x.region,
                             stable_secs=2)

but that's probably too confusing.